raiden/encoding/format.py

```diff
@@ -117,7 +117,11 @@ def __setattr__(self, name, value):
 
             length = len(value)
             if length > field.size_bytes:
-                raise ValueError('value with length {length} for {attr} is to big'.format(length=length, attr=name))
+                msg = 'value with length {length} for {attr} is to big'.format(
```

Comment:
Since we are fixing style here, let's also fix this typo.
`is to big` -> `is too big`